### ISSUE TITLE
Sample Data

### ISSUE THREAD

**Issue by charlesdaniels** [charlesdaniels/textvis#3]
The source of the data is beyond the scope of this project, but for demonstration and development purposes, data will be required to show off the capabilities of this tool.

@charlesdaniels is doing a separate project for another class that will produce suitable output

@taylorlcling had mentioned some ideas also, that might require hand populating

This issue pends on #2 for obvious reasons.

**Comment by charlesdaniels** [comment]
Here is a sample from Autocedent (this is JSON format).

[data.txt](https://github.com/charlesdaniels/textvis/files/3104917/data.txt)

**Comment by charlesdaniels** [comment]
Here's one more, also from Autocedent run on a different article.

[data.txt](https://github.com/charlesdaniels/textvis/files/3104978/data.txt)

**Comment by TaylorLCling** [comment]
As mentioned above I've been working for a while on finding, compiling, and parsing source data to truly represent the capabilities of our application. When the idea for this project was first birthed it was vaulted by Charles’s contribution of his work in CSCE 578, that generated from a .json file the means to building our larger visualization ideas that would give us a worthy sample file to present text process data. However, the sample data we had was lackluster. I volunteered to find sample data because I have spent a portion of my college career debating and organizing debates between the political groups on campus and along the way have developed a debate theory that involves an analytical approach to arguments and how they linguistically appear in debate and regular conversation. The text processing and visualization we built this project on coincidentally seemed like the perfect way to represent my theory and automate its process (even with my understanding of text processing current limitations.) The group, in the beginning, agreed that without correct sample data we would feel lackluster in our presentation. This sent me through the process of finding the right arguments, writing them myself proved to be tedious and did not seem to fit the expectations of this project which lead me to research a paper that suited the cause. What I fell upon was a beautifully crafted argument (that which I happen to disagree with) regarding the Legalization of Drugs in America, stemming from my research on the same topic for a debate a while back.  It is written by Herbert Kleber and Joseph A. Califano Jr. and appeared in *The World & I Online* (January 2006), titled **Legalization: Panacea or Pandora's Box?** I choose this document because it has clear paragraphically separations of arguments, complex but understandable sentence structure and organization that I remember truly appreciating when I first came across it. I will be using exerts from this document to perfect the sample for our presentation. Though right now I am dealing with the limitation of text processing and the hand populating will likely be much of the further work.
